website_sale_delivery_date/readme/CONTEXT.md

```
@@ -0,0 +1,47 @@
+### Example 2: Delay Type = Days
+
+**Configuration:**
+- Available delivery days: Monday to Friday
+- Delivery window: 11:00 to 18:00
+- Minimum delivery delay: 1 day
+- Earliest delivery time of day: 11:00
+
+**Results:**
+- Any order placed on Monday can be delivered earliest Tuesday starting from 11:00
+
+The sale order's commitment_date field is automatically updated based on the selected delivery date.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)